cpp/mg_utility/mg_graph.hpp

```diff
@@ -144,8 +145,10 @@ class Graph : public GraphView<TSize> {
   ///
   /// @return double weight
   double GetWeight(TSize edge_id) const override {
+    // TODO(gitbuda): GetWeight fails in the ANALYTICAL, according to the current usage it's ok to return 0, but it
+    // would be probably be better to return optional.
```

Comment:
In `community_detection_online` it seems to me there is `Update` function which should propagate changes to graph. I will have a sync with @antepusic to check what is happening on Monday and to have sync about the return value.

Reply:
@gitbuda I am wating @antepusic to return and when we sync we will update you here